Modules/Module_Processing/xASL_init_DataLoading.m

```diff
@@ -59,6 +59,12 @@
         else
             error('No rawdata or derivatives found to load subjects from, first convert DICOMs 2 BIDS');
         end
+
+        if isequal(x.opts.bProcess, [0 0 1])
+            x.opts.bReadRawdata = false;
```

Comment:
if the directory rawdata exists - it will first report above that it will read from it and then it will switch this option OFF here. That's why I put this entire IF out of this block.